src/main/java/com/github/dockerjava/core/command/ListContainersCmdImpl.java

```diff
@@ -93,10 +94,33 @@ public ListContainersCmd withBefore(String before) {
     }
 
     @Override
-    public ListContainersCmd withLabelFilter(String... labels) {
-        checkNotNull(labels, "labels was not specified");
-        this.filters.withLabels(labels);
-        return this;
+    public ListContainersCmd withNameFilter(Collection<String> name) {
+        return withFilter("name", name);
+    }
+
+    @Override
+    public ListContainersCmd withIdFilter(Collection<String> id) {
+        return withFilter("id", id);
```

Comment:
maybe make some enum for filter names?

Reply:
But for what reason?
New filters can be added by docker anytime. So where a Enum helps here?